CONTRIBUTING.md

```diff
@@ -102,7 +102,7 @@ Beyond that we have four big directories with different purposes:
 |-----------|---------|-------------|
 | **`develop/`** | This is where we _explore_ data, feel free to create your own notebook for your exploration. | `[ISO 8601 date]-[author-username]-[2-4 word description].ipynb` (e.g. `2016-05-13-anaschwendler-air-tickets.ipynb`) |
 |**`report/`** | This is where we write up the findings and results, here is where we put together different data, analysis and strategies to make a point, feel free to jump in. | Meaningful title for the report (e.g. `Transport-allowances.ipynb` |
-| **`src/`** | This is where our auxiliary scripts lies, code to scrap data, to convert stuff etc. | Small caps, no special character, `-` instead of spaces. |
```

Comment:
Actually _code to scrap data, to convert stuff_ is our _auxiliary scripts_. The enumeration after the comma was more expanding the meaning than adding stuff to _auxiliary_ script. If you think the writing is too unclear the way it is, would you mind replacing your _as well as_ by  _in other words_?